src/homework/eighth/FutureScheduleExample.java

```diff
@@ -0,0 +1,24 @@
+package homework.eighth;
+
+import java.util.concurrent.*;
+
+public class FutureScheduleExample {
+    public static void main(String[] args) throws InterruptedException, ExecutionException {
+        ScheduledExecutorService ses = Executors.newScheduledThreadPool(1);
+        Callable<Integer> task2 = () -> 77;
+        task1();
+        ScheduledFuture<Integer> schedule = ses.schedule(task2, 5, TimeUnit.SECONDS);
+        task3();
+        System.out.println(schedule.get());
+        System.out.println("shutdown!");
+        ses.shutdown();
+    }
+
+    public static void task1() {
```

Comment:
It is not an informative name. And why you need this method?

Reply:
with the help of methods task1 and task3 in this example we can see the order of tasks executing. It is shown here that task 1 and 3 in the main thread and task 2 in additional thread. We can see the output of task1 and task3 first, and later in about 5 seconds, the output of task2, so we can make a conclusion that task2, which is scheduled, does not block the main thread.